src/components/implementation/no_interface/llbooter/llbooter.c

```diff
@@ -1,5 +1,10 @@
+#include <cos_types.h>
```

Comment:
Our distinction between booter.c and boot_deps.h no longer exists, it seems.  We can discuss this after deadline.

Reply:
Yeah I think it's lost! 

Every time I write code in llbooter, this confuses me: What is the point of writing in `boot_deps.h` vs `llbooter.c`, they're just in one place and this header has evolved into being very much local to llbooter.